pkg/cmd/container/kill.go

```diff
@@ -0,0 +1,112 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package container
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"syscall"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/cio"
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/nerdctl/pkg/api/types"
+	"github.com/containerd/nerdctl/pkg/clientutil"
+	"github.com/containerd/nerdctl/pkg/idutil/containerwalker"
+	"github.com/moby/sys/signal"
+	"github.com/sirupsen/logrus"
+)
+
+func Kill(ctx context.Context, KillCommandOptions types.KillCommandOptions, stdout, stderr io.Writer, args []string) error {
```

Comment:
can you change the func signature to 

```golang
func Kill(ctx context.Context, containers []string, opt types.KillCommandOptions, stdout, stderr io.Writer) error
```

`containers []string` is the direct object `Kill` applies to, `opt` decides how to `kill`, `stdout/stderr` is mostly for output (not directly related to `Kill`)

Reply:
also `args`-> `containers` (more straightforward), `KillCommandOptions` -> `opt` (1st char shouldn't be uppercased, `opt` is shorter and straightforward)

Reply:
SGTM. it's more a nit than a change request :)